pkg/sip/server.go

```
@@ -218,7 +218,19 @@ func (s *Server) Start(agent *sipgo.UserAgent, unhandled sipgo.RequestHandler) e
 	s.sipUnhandled = unhandled
 
 	// Ignore ACKs
-	s.sipSrv.OnAck(func(req *sip.Request, tx sip.ServerTransaction) {})
+	s.sipSrv.OnAck(func(req *sip.Request, tx sip.ServerTransaction) {
+	        fromTag, err := getTagValue(req)
+	        if err != nil {
+	            sipErrorResponse(tx, req)
```

We may ignore an error here, I think.